Backend/src/authorization/validation/email-unique/email_unique_controller.ts

```diff
@@ -0,0 +1,20 @@
+import Controller from "../../../core/controller";
+import { UserService } from "../../../core/service/user_service";
+import { StatusCode } from "../../../core/codes";
+import { Request, Response } from "express";
+
+export class EmailUniqueController extends Controller {
+  /**
+   * Handles the request and response of the /email-unique route.
+   * Checks if sent email is unique.
+   *
+   * @param req :Request (express)
+   * @param res :Response (express)
+   */
+  public static async handle(req: Request, res: Response): Promise<void> {
+    const email: string | null = req.query.email as string;
```

Comment:
Wieso wird beim `req.query.email as string` das `as string` hinzugefügt. Würde nicht `req.query.email ?? null` mehr Sinn ergeben, da das Undefined so in ein NULL umgewandelt wird durch die ??.